localstack/deprecations.py

```diff
@@ -242,6 +242,13 @@ def is_affected(self) -> bool:
         "1.4.0",
         "This feature is marked for removal. Please use AWS client API to seed Kinesis streams.",
     ),
+    EnvVarDeprecation(
+        "PROVIDER_OVERRIDE_LAMBDA",
```

Comment:
Adding the provider override as a deprecation option helps users cleaning up their configuration